.github/workflows/drivers.yml

```diff
@@ -522,12 +540,30 @@ jobs:
       uses: ./.github/actions/test-driver
       with:
         junit-name: 'be-tests-mysql-8-0-ee'
+        test-args: >-
+          :partition/total 2
+          :partition/index ${{ matrix.partition-index }}
+    - name: Upload Test Results
```

Comment:
not really part of this PR but this was missing from this job even tho the other jobs all had it